interactions/models/discord/webhooks.py

```diff
@@ -256,6 +257,24 @@ async def send(
         if message_data:
             return self._client.cache.place_message_data(message_data)
 
+    async def get_message(self, message_id: Union["Message", "Snowflake_Type"]) -> Optional["Message"]:
```

Comment:
This should be `fetch_method`, to match how the library uses "fetch" for getting objects through HTTP.

(For context: the actual HTTP method is fine, since the HTTP client uses a different naming scheme than most of the library to match up with HTTP terms.)